config/jobs/kubernetes/sig-node/crio.yaml

```diff
@@ -27,10 +27,42 @@ periodics:
       - name: GOPATH
         value: /go
   annotations:
-    testgrid-dashboards: sig-node-cri-o
+    testgrid-dashboards: sig-node-cri-o, sig-release-master-informing
```

Comment:
We have to clarify this with @kubernetes/ci-signal first. There is no strict process for graduating a test to `sig-release-master-informing` from what I can see, so I propose that we create a [tracking issue in k/sig-release](https://github.com/kubernetes/sig-release/issues/new?assignees=&labels=kind%2Ffeature%2C+sig%2Frelease&template=feature.md) to reach consensus with the SIG.

Reply:
> We have to clarify this with @kubernetes/ci-signal first. There is no strict process for graduating a test to `sig-release-master-informing` from what I can see, so I propose that we create a [tracking issue in k/sig-release](https://github.com/kubernetes/sig-release/issues/new?assignees=&labels=kind%2Ffeature%2C+sig%2Frelease&template=feature.md) to reach consensus with the SIG.

Thanks @saschagrunert. cc @ehashman.

Reply:
/hold
waiting on https://github.com/kubernetes/sig-release/issues/1608